liam2/data.py

```diff
@@ -853,6 +858,11 @@ def prepare(self, globals_def, entities, input_dataset, start_period):
                 output_globals = output_file.create_group("/", "globals",
                                                           "Globals")
                 for k, g_def in globals_def.iteritems():
+                    # Do not save global constants ie those who have a value
+                    # attribute
+                    if 'value' in g_def:

```

Comment:
maybe change the test to `if 'value' in g_def or 'path' in g_def:` so that the `if 'path' not in g_def` test can be removed?